### PR TITLE
[BALANCE|NERF] Replace's Sec Borg's AdvTaser with a Disabler

### PR DESCRIPTION
**Pull request by Steelpoint**
As the title says, this PR replaces the Security Cyborgs Taser with a Disabler. 

Security Cyborgs are the most played Cyborg in the game, and the most disliked Cyborg. The recent Stun Overhaul did not affect Security Borgs aside from giving them a optional Disabler alongside their Taser, making Sec Borgs stand out as a extremely robust combat unit.

To put it frankly, as they stand Sec Borgs are among the most powerful presence on the station, easily capable of rivalling Nuclear Operatives and other antagonists, not to mention the crew.

This change means a Sec Borg has to put in slightly more effort to stun a target, instead of a one-shot stun.

For recharge times, the time to recharge the disabler from empty to full is the same as the time it took to recharge the taser, around one minute. 

Thank you.

### CODE REVIEW

**Comment by Deantwo** [comment]
Shouldn't it be "Replaces A's X **with** Y"?

Other than the missspelled title I know know much about Sec Borgs, but you do make a convising case.

**Comment by Steelpoint** [comment]
Thanks.

**Comment by Lo6a4evskiy** [comment]
This is indeed very good. Merge yesterday.

**Comment by hugelosernapkin420** [comment]
In my personal opinion, Sec borgs or any borgs for that matter need no nerfing. They have very potent weaknesses and need no more balancing. They are the only antags in game that have more than one hard counters to them which spawn on the station (Robotics console and ion rifle.) Both are basically kill switches, one literally being so. Are they powerful? Yes, of course they are, but they are also vastly outnumbered by the crew. They also are affected by the one stun and done system. Get them flashed or ioned. You can also immediately stop them by just pulling out their power cell. They have a lot of counters. No, the average crew member cannot kill them. The average crew member should not be able to kill robots. Give the average crew member flashes and then the borgs are pretty much scrap. What I'm saying is stop nerfing things just because they seem hard to kill. The reason they are hard to kill is because they are hard to kill, but can be easily stopped by a single person if they know what they are doing.

**Comment by Steelpoint** [comment]
The same thing can be asked for other low pop roles such as Security Officers, the Head of Security or the Captain.

Cyborgs have three hard counters against them at round start. 

Humans have dozens of hard counters against them at round start.

Your operating under the assumption that the entire crew is suddenly armed with Flashes or Flashbangs, immediate knowledge of the Silicon's hostile intent and the competency to form squads and attack the Security Borgs in unison. 

The fact of the matter is the crew is a disjointed mess in the best of circumstances, a single or even a small group of humans (I've seen teams of Nuke Ops and Security get dunked by a single Sec Borg) are easily countered by a Security Borg or two.

Most of the crew have no defence against a Cyborg, and even those that do have a defence will only be carrying a flash with them, the sole Ion Gun is a bulky item that must be held in your hands or put in your back slot.

I'm not saying the average crew man should be able to single handedly kill a Sec Borg, I'm saying that the average crew man should have a chance to evade a Sec Borg. The Taser does not offer this as its a one hit stun, the Disabler does offer this.

This comes down to a difference in opinions and balance, I'll let other people give their opinion on the subject and we'll go from there.

**Comment by hugelosernapkin420** [comment]
Steelpoint, have you forgotten that a major part of this game is dying with little to control over your death? You can die at anytime regardless of how well-guarded you are. That should be accepted, not swept away. Also, killing nuke ops as a borg isn't that big of a deal. They are ususally bad to average players. I remember dunking 4 out the five before the speed change, looting one, taking the pinpointer, finding the nuking and disarming it as a sec borg. I've also done that as a human (not that exact scenario, of course.) All antags (and rogue borgs are essentially antags) have a form of insta-killing people. That's why there is cloning. That's why a large part of people's defense is CHELP ROGUE BORGS. Your death ensures that people will get up in arms and try to kill the borg(s).

**Comment by Chocobro** [comment]
:+1:
also im p sure its not disliked right now, what with it currently being incredibly robust - if you're into robusting then you're gonna enjoy secborg

**Comment by Fox-McCloud** [comment]
:+1:

File: code/modules/projectiles/guns/energy/special.dm [hunk 1]
```diff
@@ -163,6 +163,36 @@
 	item_state = null
 	ammo_type = list(/obj/item/ammo_casing/energy/disabler)
 
+/obj/item/weapon/gun/energy/disabler/cyborg
+	name = "cyborg disabler"
+	desc = "An integrated hybrid taser that draws directly from a cyborg's power cell. The weapon contains a limiter to prevent the cyborg's power cell from overheating."
```
Comment (Fox-McCloud) [inline code/modules/projectiles/guns/energy/special.dm]: Needs a description change, Steel =p

**Comment by Dorsisdwarf** [comment]
I dislike this change. Sounds like someone got ambushed in their office by a secborg and is out for blood to me. 
There's a reason sec don't use their disablers except for the rare case of someone going past some glass - they're pretty bad, and while it gets easier to hit them every time (On station, that is), that's not so much consolation when you fire a shot into a perp, they slow marginally, and they unzip their ebow/flash/stunprod/baton/laser cannon/taser set to stun/flashbang/adrenal.
Giving secborgs effectively no ranged stun sort of removes the point of them, especially for actual rogue bots. After all, why bother using a disabler at all if you can just stunbaton->esword as a standard borg.
Because the situations in which you're going to be stunned and not get a call for help off before you're stripped or in crit are the situations where you're alone, not paying attention, and not aware there was a borg there. All situations in which you're going to get your ass whipped by a borg anyway.

**Comment by Lo6a4evskiy** [comment]
>Giving secborgs effectively no ranged stun sort of removes the point of them, especially for actual rogue bots.

You just described everything that's wrong with secborgs and our combat in general. This is one of the steps in the direction of fixing it.

**Comment by Steelpoint** [comment]
Thanks for the assumption, but I have not been killed or stunned by a Security Cyborg in a while. 

My reasoning for this PR is based not only on personally observation of the game for a extended period, but also from continual complaints from players about Security Cyborgs in game.

**Comment by Anonus** [comment]
You need to modify the taser cooldown upgrade module so it'll work with the new disabler module, otherwise it'll be useless.

**Comment by Steelpoint** [comment]
Done.

**Comment by paprka** [comment]
I'm down with trying this out, sec borgs being unstunnable to translate to them not being able to instastun too

**Comment by Cebasiz** [comment]
i'm not totally against this change, it's about what you want to accomplish. you said last night when you were talking about it that cyborgs are too difficult to kill and that the average crewman cannot kill a cyborg.

the problem i guess is that this isn't going to change that at all. if you think the chief problem with secborgs is that they are impossible to kill you should add more anti-cyborg weaponry, perhaps a second ion rifle or some ion pistols, so that they can be stunned and killed easier. this change, while making them in general less effective, is not going to stop them from killing an effectively unlimited amount of normal crewman, because against a normal crewman there is no disadvantage in closing to melee and beating them to death with your stun baton.

it will make them less effective at secborg V many combat of course, and make arresting people (slightly) harder. but if you think they're too immortal it isn't really going to change much in that direction.

i play sillicons probably 70% of the time, and really enjoy playing them. this change isn't really going to threaten that so it doesn't bother me, 

i guess what i'm saying is that you should nail down the problem with secborgs, and then attempt to fix that problem. while this will 'nerf' them, i'm not sure if it will fix the problem you seemed to have (the impression i got from our small chat in OOC, anyway)

**Comment by Steelpoint** [comment]
I do think making the Ion Gun in the armoury less unwieldy to use (like being able to put it in a backpack) would help the crew along. 

However I don't know the implications of giving the crew more readily available access to anti-silicon weapons. Because Ion Guns are not only ok at stopping Cyborgs, but they are godly against stopping Mechs. 

I suppose it might be interesting to investigate making the Ion Gun less cumberson, and maybe work in giving the HoS a unique firearm where one of the firing modes is a "Ion Gun" setting (the HoS was hit hard with the nerf bat in the stun overhaul).

My overall idea with this is to at least give individual crew members a small chance of outmanoeuvring a Sec Borg at range.

**Comment by paprka** [comment]
flashes fucking shit on borgs and are extremely readily available i really don't see the problem

something i suggest is removing the 'kill' or laser mode from pulse weapons and replacing it with an ion beam mode or something then you can just give the HoP a pulse pistol

**Comment by Kromgar** [comment]
Why not just give him an ionic energy pistol with the 3 setting?

**Comment by paprka** [comment]
the cap has an auto recharging laser so yeah whatever

**Comment by Steelpoint** [comment]
You'll need sprites, or just use the Pulse Pistol sprites.

Either way, that's a discussion for another thread/PR.

**Comment by paprka** [comment]
You can use the cap's cool ass laser as a base for the HoS' and make like a black/navy blue version of that with some cooler sprites but yeah that can be added later if ion stuff is really something the HoS needs or that needs to be more frequent

I just haven't seen a traitor AI without a sec borg as of late and it's very depressing. This is a good change and I think we should try it out to maybe make the sec borg less of a complete upgrade for antag hunters than playing regular officer. Silicons won't be happy but disablers were buffed pretty hard.

**Comment by Steelpoint** [comment]
I've always had a idea of letting the AI purchase better weapons for its Cyborgs with CPU power, so that even if the Sec Borg is nerfed or replaced with a Detective Borg that the AI would still be able to kit out its minions with ok weapons.

I'll investigate a HoS unique weapon after this is merged/closed.

**Comment by paprka** [comment]
I'd just go for stun/laser/ion. Pulse would do too much damage and the HoS sometimes needs controlled 20 amounts of damage rather than however much the pulse does. Pulse is a good idea for security but I plan on investigating it later.

**Comment by xxalpha** [comment]
Terrible. Adding a pistol that can do everything to your favorite role and then nerfing cyborgs because too powerful? Steelpoint, get fucking real.

**Comment by Steelpoint** [comment]
I never knew a gun with the ability to use Lasers, Tasers and Disablers was doing everything.

Security Cyborgs are a role that effect the entire crew, and people have generally held disdain for Sec Cyborgs for a long time now. In addition, many other servers have simply outright removed Security Cyborgs from their servers due to people's dislike of them.

Once again, while flashes are effective against Silicon, they are unreliable as they are a melee weapon. Even when fighting humans with a Stun Baton its very finicky to use, and if said target has a ranged weapon, and can only be stunned from a Flash or the rare Ion Gun, then things get suddenly very hard. Flashes are also a uncommon item, they are not put inside everyone's backpack at round start.

**Comment by Steelpoint** [comment]
Also try and avoid editing your posts after people have responded to them, makes it hard to keep track of things.

You can still one-shit stun people with your baton, and three disabler shots will stun a human. We can easily revert the change if its too drastic, and as Paprika said.

>I'm down with trying this out, sec borgs being unstunnable to translate to them not being able to instastun too (outside of batons)

**Comment by xxalpha** [comment]
Flashes are not unreliable. Don't get in front of a taser and you will be fine.

I will not repeat what other people have said, if you don't want to listen then don't. Turn this game into Security Station 13, no fun allowed to anyone else.

**Comment by Dorsisdwarf** [comment]
Disabler shots are lowered in damage by laser armor though, IIRC. So anything with a high laser protection takes a buncha shots

**Comment by Steelpoint** [comment]
Armour means you'll take four shots instead of three.

So only Security personal and the Captain will be affected, and they are easily distinguishable.

**Comment by Fox-McCloud** [comment]
@Dorsisdwarf 

That's not correct; they go by energy armor.

**Comment by Steelpoint** [comment]
This is merge ready, and its nearly been a week.

**Comment by Dorsisdwarf** [comment]
merge-ready =/= merge it

**Comment by GunHog** [comment]
If the HoS is going to be getting an overpowered gun that can fire 10 stun rounds, TWENTY laser rounds (that is TWICE as many as an egun), and *destroy up to five R-walls*, then this nerf is not acceptable as it shifts balance far too much in Sec's favor. Your desire to have such a weapon shows that you know how trash disablers are. If you played Secborg as much as you did HoS, would you accept this nerf?

Still a massive :-1:

**Comment by Steelpoint** [comment]
As far as I'm concerned I think that anyone who selects to play as a Security Cyborg should just be turned into a Human Security Officer, then to take responsibility for playing Security.

**Comment by Dorsisdwarf** [comment]
I play security almost as much as I play silicons. Sec needs more varied tools, which is why I'm only sorta against the gun, not nerfed foes, which is why I dislike this one.

@GunHog I was under the impression all laser rifles carried 20 rounds?

**Comment by GunHog** [comment]
Eguns have 10 laser shots, laser rifles have 12 laser shots, Steelpoint's proposed HoS gun has 20. By the way, Silicon policy is just as essay sized as Sec policy is. Human Sec officers do not have to deal with Law 1 vs Law 2 when trying to arrest someone, nor do they have trigger happy crew members ready to explode them using a remote killswitch on the smallest mistake they make.

**Comment by paprka** [comment]
GunHog maybe consider a form of balance for borgs that isn't relying on sec borgs to be instastun murder machines huh?

This isn't only about traitor antags. Normal round balance comes first, and sec borgs are power hungry antag hunters in normal rounds, just waiting to get subverted so they can instastun and kill people with extreme efficiency, about as fast as a fucking syndie borg. Pls go with your silicon apologist attitude.

**Comment by xxalpha** [comment]
>sec borgs are power hungry antag hunters

Nope. Law 1.

**Comment by paprka** [comment]
Uh, are you implying people don't ignore asimov literally 24/7 and also never get banned for being antag hunting assholes

Sec borg is simply over-equipped, for both an antag role (LOL TASE AND LASE) and an arresting role. It's not like sec borgs can be ranged stunned, so they can just rock up to people and baton them like a big fat red beepsky.

**Comment by xxalpha** [comment]
So they get banned, that doesn't warrant changing the game.

**Comment by paprka** [comment]
Did you even read my fucking post xxalpha

This has passed the point of being an administration issue. Our playerbase as a whole can't be trusted with this kind of firepower as sec borg because the assholes will pic sec borg every round and powergame, praying for antag rolls so they can tase and lase people. It's terrible.

**Comment by Steelpoint** [comment]
I should note statistically that round 50% to 60% of all Cyborgs selected are Security Cyborgs. A clear indicator that the Sec Borg is the go to Cyborg.

**Comment by xxalpha** [comment]
The assholes can easily be banned if they break the laws, the rest of us shouldn't have to suffer for it. Since flashes are now useless to stop humans, the taser is the only thing available to security cyborgs to stop harm immediately. If you want to replace the taser with a disabler, make the cyborg disabler only take two shots to fatigue crit a person, armored or no.

**Comment by paprka** [comment]
Useful =/= powergame

Medborg is more useful to the crew than secborg ever will be

Secborg is powergamer wish fulfillment

**Comment by Steelpoint** [comment]
On occasions where security cyborgs have rendered assistant to security, it has been greatly useful.

Sadly most of the time, from my own personal experience, security cyborgs typically fall into the following categories

* They are too busy chasing Changelings/Traitors at the direction of the AI. (Its depressing to hear robo chat when this happens)
* They arrest people for inane things and refuse to compromise or be swayed, even if sec tells them otherwise.

To be frank I think Sec Borgs should either be removed or replaced with a Detective Borg, but this is a fair compromise. If anyone has played any rounds with a subverted/malf AI they'll know the insane bull that Sec Borgs can pull.

--

Again, I have seen entire teams of Nuke Ops, and Security, get wiped out from a single Sec Borg.

Saying "git gut" is not good enough, that's like telling a assistant to get good against a Deathsquad Officer. It ain't going to happen.

A disabler is a fair compromise, sec borgs are still capable of one-hit stuns with their Stun Batons, but they have to have some skill/effort to stun at range.

Hell, this might even encourage people to use the Standard module since it gets a baton.

**Comment by xxalpha** [comment]
>Secborg is powergamer robocop wish fulfillment and impedes security more often than not

Robocop is bannable behavior, ahelp.

All the reasons being presented is people not following their laws. There's already a solution for that, they get banned from silicon.

**Comment by Steelpoint** [comment]
> Ignoring my reasons

The behaviour I've outlined that many Sec Borgs partake in is not against the rules.

**Comment by Dorsisdwarf** [comment]
If the sort of assistant we tend to get can't take out a deathsquaddie who's alone, then I'd be suprised. 
And an entire team of nuke ops who are so bad and so unorganised that a secborg can take them and their mandatory syndieborg out (Epecially since nukies get an ion rifle) without being flashed, crowbarred, and emagged, pretty much deserve to die.

People play secborg and engiborg because they're the only roles which are any fun and which can do anything proactively. Waiting half an hour as a medborg for people to get hurt and not be dead and then getting bwoinked for emptying your poisonous medipen into them (Who thought that was a good idea ever) isn't fun.
Janiborg isn't much fun except to run circles around the janitor/be rogue and lube the entire universe.
Service borg is a joke.

Saying "Sometimes you WILL die" is a fair point. Next it'll be removing mute pens and stun weapons from traitors because they can also shit all over a sec team if the player is competent.

Generally the reason that secborgs are able to robust entire stations is when the players playing them are the ones who are really good at it. Why are the borgs usually more competent at fighting than sec? Because borgs are hard to play and easy to get banned from. Sec officer is easy to run as long as you arent permabrigging for minor crimes or executing sans license. Only the decent players tend to /survive/ long as borgs.

Also let's conveniently forget all the times that sec spots a malf APC/glimpses the borg harmbatoning someone and scootles over and tells the R&D to blow the borgs GG no re, silicons.


And a secborg validhunting human traitors is shit,  and ignoring orders from nonharmful crew to let it go IS ignoring their laws and CAN be banned.

**Comment by Steelpoint** [comment]
Ops rarely take the Ion Rifle because its unwieldy and it cannot fit in their backpack. I'm going to stop trying to write essays, so I'll leave my argument as this.

Cyborgs, who are unstunable by most metrics, should not have a ranged instant stun attack.

This is a minor code change that can easily be reverted if its found that this was too server a nerf to Silicons, that or by giving emagged non-sec borgs access to better equipment.

**Comment by xxalpha** [comment]
Cyborgs are easily stunnable with a flash (and I think ion guns stun?). They can also be all locked down remotely. Security borgs are the only ones with any stunning capability when not hacked, because they're fucking security cyborgs.

**Comment by Steelpoint** [comment]
Humans are stunned by, in no order...
* Stun Attacks (Ranged or Melee)
* RNG hits from Eswords
* RNG hits from Armblades
* EMP overload
* Chemical attacks
* Disable Attacks
* Explosions
* RNG disarm attacks
* Sleepers/Mechs
* Being put into critical (even in crit a Cyborg can more!)
* Breathing in sleeping gas/posionus air in general
* Capsicum
* Being thrown on a table
* Being thrown on a table by the server host
* Wall mounted flashers/mobile flashers
* Ballistic guns with a high weaken value (c20, syndi shotgun)
* Hallucinations
* Being on fire/resisting
* Being in handcuffs counts.
* More

Cyborgs are stunned by...
* Flash/Flashbangs
* Iongun
* Robotics Console

**Comment by Dorsisdwarf** [comment]
I like how you artificially padded your list by repeating articles. 
Being in handcuffs isn't a stun.
Disable attacks are the same thing as weakening guns.
Chemical attacks, capsicum, being flashed, breathing in poison gases/mobile flashers and the like can all be blocked with easy to obtain gear such as flashes and gas masks (cough cough you took those off of sec)
Eswords, armblades, the explosions, are all knockouts.
Being put into crit is not a stun, it's being put into crit. Go ahead, crit the next assistant you see as sec with a harmbaton then tell the adminbwoink that you were "just stunning him".
The only thing a disarm will do (to people not carrying a riot shield/dual swords that is) is knock them over for half a second and let you steal their gun if they had it out.

In addition, borgs can be taken out of action by:
Laser laser laser
smash smash smash and watch them lose their modules
Laser pointer
Running out of power (serious threat for borgs spamming batons) and being effectively straightjacketed and muted until someone plugs them back into a charger
EMPs from chem
EMPs from xenobio
EMPs from mining
EMPs from traitors
EMPs from changelings
EMPs from the EMP grenades in space
The poliz baton
Maybe even the telebaton, but don't hold me on that one.


Silicon Rogue is probably overpowered as it is, but nerfing the AI's main offensive tool and at the same time giving the HOS what is effectively an unstoppable kill on the AI is a liiiiitle much, eh?

**Comment by paprka** [comment]
Half of those things you listed are things nobody will ever make under normal circumstances.

"EMPS from_" can all be listed as one thing. EMPs. Which are hard as fuck to obtain.

Borgs are still vastly superior and can tank a metric fuckton of damage, maybe we should make them more frail or something considering they can be literally instantly revived.

**Comment by Steelpoint** [comment]
The other half are not stuns, simply killing the Borg.

**Comment by Dorsisdwarf** [comment]
They can be instantly revived.... By robotics. Who are the enemy. And once they're revived by robotics, the rogue AI is dead orthe traitor is screwed.

And I included a vast list of every source of EMP because steelpoint listed "chemical warfare", "Caspacin", and "poisons" as different stuns.

**Comment by Dorsisdwarf** [comment]
Gee, I never knew emping/laser pointering were "killing the borg" and "not stuns".

The police baton stuns everything. Except maybe xenos.

**Comment by Steelpoint** [comment]
Look, I'm just replacing the AdvTaser with a Disabler, I'm not killing Jesus.

**Comment by Dorsisdwarf** [comment]
I'm not even particularly against this change in itself. I'm more against your opinion that sec should be able to shut down the immobile rogue AI super easy and that borgs are gods of war as it is and need to be put in the naughty corner.

I'd support making their stuns more powerful, but giving the HOS a tool which literally lets him execute the AI at will and taking away the taser from borgs seems ridiculous.

**Comment by Steelpoint** [comment]
Then kill the HoS, its surprisingly easy. Blow up his console or laser him to death. I would know, people try that on me all the time. If the HoS can survive all that, then he's just a good player.

**Comment by Dorsisdwarf** [comment]
So your new proposal mandatory strategy for malf/subverted AI is kill the HOS at roundstart every time and space his Win Gun to prevent him from blasting into the armory, grabbing a suit, lasering out the window with his spare egun and spacewalking to the sat where he casually executes the AI, maybe stopping to weld the doors shut so secborgs can't get in if he feels slow?

**Comment by paprka** [comment]
HoS with all of his tools will never get out of a bolted room with no windows that is depowered. He isn't the CE. It's extremely easy to take out the HoS. If you rely on your secborg having stuns you are terrible.

**Comment by Dorsisdwarf** [comment]
It had better be a room with no intercomm, he'd better not have an SBR, and you'd better have muted him completely from comms/disabled comms, or it's "AI malf" for you.

**Comment by paprka** [comment]
so your only complaints are you can't take down a HoS steathily or he might cry AI ROGUE?

seriously, are you just fucking retarded or what? why would you need to take down a HoS unless you planned on going loud in the first place? no you're right, the HoS should be a complete pushover like every other crewmember, and you should have borgs that instastun people which is effectively complete game over once they tase and lase.

10/10. you trolled me, I almost thought people were THAT RETARDED to think tase & lase is good gameplay or effectively balanced.

Borgs don't need tasers. I did the stun overhaul. I should have done this in the first place. Beepsky arrests scumbags with no ranged weaponry. Secborgs just need to get good. Secborgs can also just like laser people into crit if they're antags instead of trying to be ezmode and stun them first, but I don't expect anyone who plays sec borg to know anything about combat in ss13 lol

**Comment by Dorsisdwarf** [comment]
Resorting to ad homenim person attacks to hide the fact that you want to nerf secborgs ( And therefore malf AIs and rogue AIs and subverted AIs and purged AIs), and in addition give the HOS a gun that /literally lets him casually execute the AI unless you fill your core with airlocks/ is childish

**Comment by Dorsisdwarf** [comment]
Wait sorry forgot I was talking to paprika here

**Comment by Steelpoint** [comment]
The whole point of this PR is a nerf, its even in the title. I am not hiding that fact.

**Comment by Dorsisdwarf** [comment]
Don't mix up my criticism of this PR, my criticism of the two PRs together, and my criticism of pap being an ass.

On it's own, I think that it would be pretty neat to see how borg strategy changes. With the breacher gun, I think it's too much down the other way of the sec/AI seesaw.

**Comment by paprka** [comment]
I don't really care about the HoS' personal gun in relation to this PR

This is about tase and lase, a feature that is almost nill with security now due to tasers having low ammo count and needing to be manually recharged, and that eguns have had their taser feature removed, whereas borg tasers recharge automatically

I would prefer if the HoS DIDN'T have a snowflake gun, but that PR wasn't originally about AIs anyway. Ion shots can be used for a lot more than just killing borgs, and if breacher shots are really so concerning to you, we can make them simply not destroy r-walls. The point behind breach mode with the HoS' gun is so that he can get to criminals who lock doors. Relying on the AI to open doors when sec doesn't have access to an area is bad, the HoS should be virtually unstoppable in this regard honestly. After all, sec borgs have all access, are unstunnable, and the only thing that can stop them is a clunky ion rifle or a rare EMP grenade/rng laser pointer.

**Comment by Jordie0608** [comment]
Changelog.

**Comment by Steelpoint** [comment]
Done.

**Comment by Dorsisdwarf** [comment]
There really is a difference between 'lel silicon defense squad' and 'lel make the AI a pushover'.

As it is, secborgs are too effective. That's a certainty. It needs rebalancing.
Merging this change and the current iteration of the HOS firearm pushes it too hard the other way, and actively forces malf/rogue AIs to move their core/powergame with airlock barricades in order to not be annihilated within a minute of going loud.

**Comment by Steelpoint** [comment]
How do we know that? Sec Borgs, and most Cyborgs, will still have reliable weapons. AI's can still cut comms, blow up computers, flood plasma, fortify their core, etc.

AI's wining malf have always been about "powergaming" as hard as possible one way or the other.

**Comment by paprka** [comment]
I don't really care what you think dorsisdwarf, but I removed tase and lase, antagonist borgs are no exception. Their tasers automatically recharge and so do their laser guns. It's basically infinite murderbone. Borgs do not need insta range stuns, because they have significantly less sources to get instantly range stunned by. Sec officers can't take this risk or they get murdered and all of their gear gets taken, which is why they have a taser instead of a disabler right now, because antags still have access to stuns (and i don't plan on changing that quite yet.)

Borgs have nothing to fear from being beepsky and rocking up to people and stunbattoning them. There is absolutely no reason borgs need ranged stuns. 3 disabler hits on most targets weakens them, and if someone has an ion rifle... Don't engage them directly. Because that's literally how fucking counters work in game balance. It would be like taking on an anti air tank with a helicopter.

**Comment by Dorsisdwarf** [comment]
Which is why I suggested cutting their taser shots, maybe even down to just 1/2, so that they retain their ambush-ness, while making them unable to down a whole crowd of security. if you've wandered off from the rest of the team as a nuke op/ sec officer during malf when there's borgs about, you pretty much deserve to be stunned GG. Malf's a many-vs-few gametype, after all, and should retain a sense of having to stick and work together.

**Comment by paprka** [comment]
"so that they retain their ambush-ness"

nope, this is shit gameplay and 1 hit gg no re. they should at least require being adjecent (IE, stun baton)

**Comment by xxalpha** [comment]
How is that shit gameplay, that's literally what sec does.

**Comment by paprka** [comment]
Sec can't be antagonists

**Comment by xxalpha** [comment]
Neither can Asimov cyborgs.

**Comment by paprka** [comment]
Xxalpha confirmed for literally brain dead.

**Comment by Steelpoint** [comment]
If you honestly think that xxalpha then you've clearly not played the game often, are being wilfully ignorant or something else that eludes me.

**Comment by xxalpha** [comment]
I only play cyborg half the fucking time. This whole "cyborgs can't be stunned so they are not allowed to have stuns" shit is ridiculous. They can be stunned and killed far more easily than humans can and only the secborg has a taser.

**Comment by Steelpoint** [comment]
>stunned far more easily than humans

Your objectively incorrect, go up a bit to find my big list of things that stun humans versus things that stun cyborgs.

**Comment by xxalpha** [comment]
Just the cyborg computer outweights everything else. Then you have ion guns, flashes and flashbangs.

Cyborgs are asimov, period. If something goes wrong silicons, it's expected that they become a threat. This is not a singleplayer game where the objective is for the spess marine to kill all badies, this is a game where shit hits the fan and everybody can lose. If you can't handle losing then stop playing.

**Comment by paprka** [comment]
I think every advanced taser should have an ion mode. Maybe then cyborg nerds can feel the butthurt.

**Comment by paprka** [comment]
[04:39] <xxalpha> then do this:
[04:39] <xxalpha> when hacked, the taser is replaced with a laser
[04:39] <xxalpha> problem solved
[04:39] <xxalpha> asimov cyborgs keep their taser

do this

**Comment by Dorsisdwarf** [comment]
If you're so salty about being tased ever pap, why not just go the full hog and remove hard stuns entirely from the game?

That's not a jab/making fun/being rude, by the way, it's an actual suggestion. It'd be genuinely interesting to see.

**Comment by SconesC** [comment]
Highly disagree but the overpowering hate against Secborgs seems to be the driving force behind this

Getting killed by a borg feels like bullshit, and getting killed as a borg feels like bullshit, all because of the incredibly long stun times on certain things (Especially a flash, holy shit)

**Comment by paprka** [comment]
I really don't think secborg player salt should be the catalyst behind us moving away from one hit stuns entirely. The difference between me and ergovisavi is that while we both have the same end goal of making combat more fair for everyone, a near impossible mission, he wanted to remove all stuns and tell the players to deal with it. I have a different design philosophy when it comes to games. I gave borgs an advanced taser instead of just removing their taser and replacing it with a disabler because I wanted to see how it would turn out. Combat with borgs is heavily skewed towards the sec borg when it comes to fighting humans because of their taser.

This isn't as much of an issue for non-antag borgs. You shouldn't have to worry that an unstunnable machine is going to pop you with a taser at any point and could be rogue. Sec borgs are the number one driving force behind AI IS ROGUE paranoia because of their ability to drop people so easily. This is not good gameplay. This is not me saying 'sec borgs shouldnt be effective antags'. Sec borgs are never antagonists themselves, but they are very wise decisions for antagonists to both subvert or choose as modules when you're malf.

If you don't see the implications here about gameplay, you're just a fucking moron and I'm not going to entertain a discussion with you anymore. That's all there is to it. It's basic fucking kiddie game balance. Secborg loadouts as antags are unfair as fuck. Personally, I don't think they should even have them as non-antags, because they can easily stun baton criminals now that flashes are so worthless that nobody really carries them anymore.

**Comment by Dorsisdwarf** [comment]
So the easy-to-obtain hard counter against silicons isn't really a hard counter because they're less effective against non-silicons?

I honestly can't tell if you're serious that having the ability to stun and kill rapidly (Even though it IS overpowered and it IS being removed somewhere or else) is utterly unique, or unfair. Security can tase and harmbaton. The HOS can tase and lase. The captain can tase and lase if he grabs a taser from the armory. Traitors can ebow-lase. Hell, they can crit you from offscreen in less than half a second with their revolver, and that's even less fun than being stunned and critted by a borg, because at least you'll get a chance to scream ;borgs rogue help when the laser comes out.

Secborgs aren't the "driving force behind AI IS ROGUE paranoia", people who scream it whenever anything doesnt go their way are.

**Comment by paprka** [comment]
GEE SOMEONE HAS AN EASY TO OBTAIN FLASH THAT CAN BREAK AT RANDOM AND REQUIRES YOU TO BE ADJACENT

NONE OF YOUR LISTED JOBS, SEC, CAP, ETC, CAN BE ANTAGONISTS. HOW ARE YOU THIS STUPID.

**Comment by xxalpha** [comment]
Antags are deadly dangerous, what else is new paprika? Yes, they might at any time be subverted and wreck your shit, but if you find a secborg being suspicious you have reason to investigate and you should investigate. But the element of surprise is part of subverted silicons / rogue AIs.

Removing a useful thing just because it will give you better chances of winning is not valid.

**Comment by Dorsisdwarf** [comment]
I really, honestly, don't see where paprika is coming from. At all. There's nothing but aimless screaming and deliberately missing the point.

**Comment by paprka** [comment]
Borg players can't see what the problem is with borgs being overpowered, news at 11

**Comment by xxalpha** [comment]
Non borg players can't understand why borg players don't need to be nerfed, news at 10.

**Comment by Steelpoint** [comment]
Security Cyborgs are among the most complained against antagonists/hostile force in the game currently. They are very powerful and that cannot be denied, not to mention that Sec Borgs are the most played module in the game.

People have wanted to get rid of Sec Borgs for a long time, and several other servers have done just that.

All this PR does is attempt to tone down the power of Sec Borgs without removing them, which I think you'll agree is preferable to gutting Sec Borgs.

**Comment by xxalpha** [comment]
I don't see any complaints at all. They must be in the deadchat where all the dead people angrily complain about being killed.

**Comment by paprka** [comment]
thank you captain obvious

**Comment by xxalpha** [comment]
People don't make the right decisions when they're angry.

**Comment by Steelpoint** [comment]
Generally people are more annoyed by dying to Sec Borgs because its unavoidable, you can't close a door on it, you can't disarm it, flashes a very uncommon item (despite what you want to think) and you can't really fight it.

A human antagonist can be avoided better, he can be disarmed, you can close a door on him, you can fight him.

**Comment by paprka** [comment]
Fuck it I'm removing all the ranged stuns at this point they're bullshit. Even if it makes security harder at least we won't have instastun ebows and tase and lase ebows. Shit is pissin me off no.

**Comment by xxalpha** [comment]
There are a lot of things that are unavoidable and kill you, Steelpoint. Secborgs armed with tasers and lasers are not the worst of it.

**Comment by Steelpoint** [comment]
If your going to do that, buff pepper spray to work even if a target is wearing sunglasses.

**Comment by Dorsisdwarf** [comment]
It does, IIRC. Just not as well.

**Comment by paprka** [comment]
Pepperspray does work if eyes are covered, but the mouth needs to not be covered as well. There's different effects depending on what part of the face is covered, but it's not really that good if you hit people who have eyes covered. I might rebalance it as well.

Syndie borgs do not have a taser, they have a very limited ammo grenade launcher that can stun but only a couple times before running out and it can't be reloaded.

**Comment by Steelpoint** [comment]
If you have sunglasses your vision simply gets foggy akin to wielding stuff to much.

**Pull request merged**